packages/frontend-shared/src/graphql/testUrqlClient.ts

```diff
@@ -2,10 +2,12 @@ import { Client, createClient, dedupExchange, errorExchange } from '@urql/core'
 import { executeExchange } from '@urql/exchange-execute'
 import { makeCacheExchange } from './urqlClient'
 import type * as stubData from './testStubCloudTypes'
+import type { navigationMenu } from './testNavigationMenu'
 import { clientTestSchema } from './clientTestSchema'
 
 export interface ClientTestContext {
   stubData: typeof stubData
+  navigationMenu: typeof navigationMenu
```

Comment:
The remaining check-ts failures are from app and wizard missing from this context. I wasn't sure if we wanted to create them like this or if we had a different plan like the stubdata.

Reply:
Yeah, go ahead and create them there.